erpnext/selling/doctype/sales_order/test_sales_order.py

```diff
@@ -136,6 +136,31 @@ def test_make_sales_invoice_with_terms(self):
 		si1 = make_sales_invoice(so.name)
 		self.assertEqual(len(si1.get("items")), 0)
 
+	def test_make_sales_invoice_with_individual_terms(self):
```

Comment:
Same as on the purchase side: see if individualized Payment Terms are preserved.